src/plone/api/tests/test_group.py

```diff
@@ -480,17 +481,27 @@ def test_grant_roles_in_context(self):
         ROLES = set(['Editor', 'Contributor'])
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(
-                groupname='foo', obj=folder, inherit=False)),
+            set(
+                api.group.get_roles(
+                    groupname='foo',
+                    obj=folder,
+                    inherit=False,
+                ),
+            ),
         )
         self.assertEqual(
             ROLES,
-            set(api.group.get_roles(group=group, obj=folder, inherit=False)),
+            set(api.group.get_roles(group=group, obj=folder, inherit=False,)),
```

Comment:
I agree, that not what we want. 

I also consider the plugin flake8-commas as broken. [flake8-commas](https://pypi.python.org/pypi/flake8-commas) states "Now you can avoid those annoying merge conflicts on dictionary and list diffs." But it also takes parameters into account, which is more than it should.

I am +1000 to disable the plugin until it works.